### ISSUE TITLE
Update project infomation

### ISSUE THREAD

**Issue by andreas-bok-sociomantic** [sociomantic/libdrizzle-redux#13]
How should [**COPYING**](https://github.com/sociomantic/libdrizzle-redux/blob/master/COPYING) and [**AUTHORS**](https://github.com/sociomantic/libdrizzle-redux/blob/master/AUTHORS) be updated after 
the forking of the official `libdrizzle-redux`

Current license is `Simplified BSD Licence`

Current authors are:

```
Brian Aker <brian@tangent.org>
Eric Day <eday@oddments.org>
Andrew Hutchings <andrew@linuxjedi.co.uk>
```

**Comment by leandro-lucarella-sociomantic** [comment]
I think we can add a Copyright (C) 2015-2016 Sociomantic Labs GmbH in the COPYING file and your name in the AUTHORS file. Here is a related stackoverflow question: http://programmers.stackexchange.com/questions/114701/what-are-the-legal-considerations-when-forking-a-bsd-licensed-project

Whatever we do, I would send an e-mail to the previous authors to see if they see anything wrong with our approach.

**Comment by andreas-bok-sociomantic** [comment]
Related PR closed #15
